src/test/java/ebl/v2/ShipmentEventIT.java

```diff
@@ -27,14 +33,18 @@ static void configs() throws IOException {
   }
 
   @Test
-  void testGetAllEvents() {
+  void testGetAllEventsAndHeaders() {
     given()
       .contentType("application/json")
       .get("/v2/events")
       .then()
       .assertThat()
       .statusCode(200)
       .contentType(ContentType.JSON)
+      .header("API-Version", equalTo("2.0.0"))
+      .header("Current-Page", matchesRegex("^http://.*/v2/events\\?cursor=[a-zA-Z\\d]*$"))
+      .header("Next-Page", matchesRegex("^http://.*/v2/events\\?cursor=[a-zA-Z\\d]*$"))
+      .header("Last-Page", matchesRegex("^http://.*/v2/events\\?cursor=[a-zA-Z\\d]*$"))
```

Comment:
I would prefer `https?` (I know it is unlikely to matter for the IT for eBL, but the pattern will likely be copy-wasted many places, so it seems like a small price to future proof it)

Reply:
Good point